src/sa_web/static/js/views/place-list-view.js

```diff
@@ -244,7 +246,7 @@ var Shareabouts = Shareabouts || {};
 
         // If we've fallen through here, hide the item.
         return hide();
-      });
+      }, this);
```

Comment:
It doesn't seem like any of the changes in this file does anything. If these changes have no effect, can we remove them?